molecule/docker/molecule.yml

```diff
@@ -7,14 +7,14 @@ lint:
   options:
     config-file: tests/yamllint.yml
 platforms:
-  - name: wipefs
-    image: centos:7
+  - name: disk_wipe
+    #image: centos:7
     ## Remove 'image' above and uncomment the following to run systemd in Docker
-    # image: centos/systemd
-    # privileged: true
-    # command: /usr/lib/systemd/systemd --system
-    # volumes:
-    #   - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    image: centos/systemd
```

Comment:
I'm hazarding a guess that we can't actually wipe disks with Travis CI/Docker.  In that case, is there any reason to use systemd here?

Reply:
fixed